alarmee/src/androidMain/kotlin/com/tweener/alarmee/AlarmeeSchedulerAndroid.kt

```diff
@@ -107,6 +107,37 @@ class AlarmeeSchedulerAndroid(
         }
     }
 
+    override fun pushNotificationNow(alarmee: Alarmee, onSuccess: () -> Unit) {
+        createNotificationChannels(context = context)
+        validateNotificationChannelId(alarmee)
+
+        val notification = alarmee.androidNotificationConfiguration.channelId?.let {
+            NotificationCompat.Builder(
+                context,
+                it
+            )
+                .apply {
+                    setSmallIcon(configuration.notificationIconResId)
+                    setContentTitle(alarmee.notificationTitle)
+                    setContentText(alarmee.notificationBody)
+                    setPriority(mapPriority(alarmee.androidNotificationConfiguration.priority))
+                    setAutoCancel(true)
+                }
+                .build()
+        }
```

Comment:
Note for myself: This will need to be refactored to avoid code duplication with `NotificationBroadcastReceiver` but for the scope of this PR, this is okay.